binder/requirements.txt

```diff
@@ -1,12 +1,7 @@
--e .['examples']
+-e .[all_pip,build]
```

Comment:
binder did not work because non-pip dependencies where added to the `examples`.

Personally I think the packages in `extras_require` should be `pip` installable. Conda and other packages should be somewhere else.  `extras_require` is a `setuptools`/ `pip` thing and should be used for that.

Reply:
> Personally I think the packages in extras_require should be pip installable. Conda and other packages should be somewhere else. extras_require is a setuptools/ pip thing and should be used for that only.

Yeah that's a bug.

Reply:
Once the hatch stuff is fully worked out they won't have to be listed at all because we will automatically map `pip` dependencies to `conda`.

Reply:
I tracked down the graphviz stuff today and decided to remove all of it anyway in the hatch PR. The graphviz example is gone and the inheritance diagrams in the API docs aren't very useful. I just hope it's not used in other places, I'll see what breaks if it does.

This is where `python-graphviz` was added to the extras: https://github.com/holoviz/panel/pull/2732/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R157

Reply:
Great!